pantheon-bundle/src/main/java/com/redhat/pantheon/extension/HydraIntegration.java

```diff
@@ -23,19 +22,27 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.redhat.pantheon.extension.events.ModuleVersionPublishStateEvent;
 import com.redhat.pantheon.extension.events.ModuleVersionPublishedEvent;
```

Comment:
I'm having a bit of trouble following these event types. Could you explain what the difference is between `ModuleVersionPublishStateEvent` and `ModuleVerisonPublishedEvent`? they seem to be the same thing

Reply:
line 25 is a result of Eclipse reorganizing its import statements. It used to be on line 15.
I think ModuleVersionPublishStateEvent can be a ModuleVersionPublishedEvent or ModuleVersionUnpublishedEvent